src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,7 +54,8 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInNameLowerCase = Utils.convertStringsToLowerCase(wordsInName);
```

Comment:
Lines 56, 57 can be changed to one line. For e.g.
final Set<String> wordsInNameLowerCase = new HashSet<>(person.getName().getWordsInName().toLowerCase());